src/python/grpcio_tests/tests/_sanity/_sanity_test.py

```diff
@@ -39,9 +39,9 @@ def testTestsJsonUpToDate(self):
         )
 
         tests_json_string = pkgutil.get_data(self.TEST_PKG_PATH, "tests.json")
-        tests_json = tests_json_string.decode()
+        tests_json = json.loads(tests_json_string.decode())
 
-        self.assertSequenceEqual(tests_json, test_suite_names)
+        self.assertListEqual(tests_json, test_suite_names)
```

Comment:
Let me make sure I understand what the exact failure mode here was. This test looks like it _should_ fail. Is the issue that this test wasn't actually running at all due to the same test discovery issue?

Reply:
`assertListEqual` is fine. I just want to make sure I understand what was happening here.